constants.py

```diff
@@ -0,0 +1 @@
+UPDATE_DELAY = 86400 #24hrs
```

Comment:
24 hours in milliseconds (make the comment more clear), also there should be a space after the `#`, so `# 24`